Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.MixedReality.Toolkit.Internal.EventDatum.Boundary;
 using Microsoft.MixedReality.Toolkit.Internal.Interfaces.BoundarySystem;
 using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities;
```

Comment:
They showed up when I did a merge with the latest mrtk_development branch.

Reply:
I think the best way to not have this happen is this:
1) Always keep your fork's mrtk_development free of any changes.
2) When you want to update, merge Microsoft's mrtk_development into your fork's, and it will automatically fast forward with no merge to deal with. Beautiful clean.
3) Create your changes in a branch based on your fork's mrtk_development, don't ever* merge into it.
4) Whenever you get an update from Microsoft's fork, *rebase* your branch to the new tip of your fork's updated mrtk_development branch.

Reply:
Yep, that's what I followed, not sure why all of this stuff showed up. I'll try to revert and see what I can do.